CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+How to contribute
+=================
+
+Thanks for your interest in contributing to Kinto!
+
+## Reporting Bugs
+
+Report bugs at https://github.com/Kinto/kinto-http.py/issues/new
+
+If you are reporting a bug, please include:
+
+ - Any details about your local setup that might be helpful in troubleshooting.
+ - Detailed steps to reproduce the bug.
+
+
+## Ready to contribute?
+
+### System Requirements
+
+Depending on the platform and chosen configuration, some libraries or extra services are required.
+
+The following commands will install necessary tools for cryptography and Python packaging like Virtualenv.
+
+#### Linux
+On Debian / Ubuntu based systems:
+
+```bash
+apt-get install libffi-dev libssl-dev python-dev python-virtualenv
+```
+
+#### OS X
+Assuming [brew](http://brew.sh/) is installed:
+
+```bash
+brew install libffi openssl pkg-config python
+
+pip install virtualenv
+```
+
+
+### Getting Started
+
+ -  Fork the repo on GitHub.
+ -  Clone your fork locally:
+
+```bash
+git clone git@github.com:your_name_here/kinto-http.py.git
+```
+
+
+## Testing
+
```

Comment:
- ``make tests-once`` to run the test with the current venv.
- ``make tests`` to run all the tests (with Py2 and Py3, flake8 and functional tests)

You may need to use ``make run-kinto`` before running the functional tests.
If you want to run the functional tests only, you can use ``make functional``